sympy/polys/polytools.py

```diff
@@ -2669,6 +2676,140 @@ def subresultants(f, g):
 
         return list(map(per, result))
 
+    def subresultant_polys(f, g):
+        """
+        Computes the subresultant polynomials of two polynomials ``f`` and ``g``.
+        The returned list has length min(deg(f), deg(g))+1, and is in order of
+        increasing degree. The first polynomial in the list is the resultant.
```

Comment:
The first line of the docstring should be a single line sentence as the one line summary

https://docs.sympy.org/latest/contributing/docstring.html#single-sentence-summary

I would say here that it should be:
```
Subresultant polynomials of ``f`` and ``g`` with respect to the main variable.
```